src/components/RepoDetails/RepoDetailList.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import RepoDetail from "./RepoDetail";
+
+const RepoDetailList = props => {
+  const { repos } = props;
+  repos.sort(function(a, b) {
+    return b.stargazers_count - a.stargazers_count;
+  });
+
+  return (
+    <div>
+      {repos.map(repo => (
+        <RepoDetail
+          key={repo.id}
+          name={repo.name}
+          stargazers_count={repo.stargazers_count}
+        />
+      ))}
+    </div>
+  );
+};
```

Comment:
```suggestion
  return repos.map(repo => <RepoDetail key={repo.id} name={repo.name} stargazers_count={repo.stargazers_count}/>);
};
```
이렇게는 안될까요??

Reply:
오옹!! 훨씬 더 보기 간편하네요ㅎㅎ return을 항상 저런식으로 하다보니까 똑같이 맞췄던것 같아요... 감사합니다&#128077;

Reply:
props가 2개 이상이 되거나 안에 내용이 길어져서 한 눈에 보이지 않는다면 줄 바꿈이 있는게 가독성이 더 높지 않을까요??

Reply:
저도 Yuni-Q님 말씀처럼 여러개의 props가 있고 내용이 길어져, 줄바꿈 하는 것이 좀 더 가독성이 높지 않을까 생각했습니다.

Reply:
소소한 tip이라면 아래와 같이 구조분해 해서 사용할 수 도있을 것 같아요

```suggestion
  return (
    <div>
      {repos.map(({id, name, stargazers_count}) => (
        <RepoDetail
          key={id}
          name={name}
          stargazers_count={stargazers_count}
        />
      ))}
    </div>
  );
};
```